locales/en.yml

```diff
@@ -70,3 +70,22 @@ en:
       description: Discover the story of our Logo
       page_title: Logo
     title: Logo of Ruby Nord
+  talks:
+    description: |
+      Find all the past talks from our Apéros Ruby.
+      Whether you're a beginner or a confirmed developer, feel free to give a talk and share your experience with the community.
+    level:
+      all: all levels
```

Comment:
* `all` stands for any level
* `soft` stands for soft skill aka not a tech talk

What about using `soft skill`? And what would be the :fr: translation?
Sometimes French conferences don't translate it.